src/components/molecules/InternalChevronAccordion/InternalChevronAccordion.tsx

```diff
@@ -100,6 +100,7 @@ const Accordion = ({
             transform: isOpen ? "rotate(180deg)" : "none",
             transition: "all 0.3s ease 0s",
           }}
+          $mr={"space-between-xs"}
```

Comment:
this only affects `OakOutlineAccordion` where the change is barely visible so I decided to keep it here rather than add more props to the `InternalChevronAccordion`